esphome/components/mitsubishi_itp/muart_packet.h

```diff
@@ -88,23 +91,23 @@ class ConnectResponsePacket : public Packet {
 };
 
 ////
-// Extended Connect
+// Identify packets
 ////
-class ExtendedConnectRequestPacket : public Packet {
+class BaseCapabilitiesRequestPacket : public Packet {
  public:
-  static ExtendedConnectRequestPacket &instance() {
-    static ExtendedConnectRequestPacket instance;
+  static BaseCapabilitiesRequestPacket &instance() {
+    static BaseCapabilitiesRequestPacket instance;
     return instance;
   }
   using Packet::Packet;
 
  private:
-  ExtendedConnectRequestPacket() : Packet(RawPacket(PacketType::EXTENDED_CONNECT_REQUEST, 1)) {
+  BaseCapabilitiesRequestPacket() : Packet(RawPacket(PacketType::IDENTIFY_REQUEST, 1)) {
     pkt_.set_payload_byte(0, 0xc9);
   }
 };
 
-class ExtendedConnectResponsePacket : public Packet {
+class BaseCapabilitiesResponsePacket : public Packet {
```

Comment:
`BaseCapabilities` was chosen since packet CD (and D0 for our JP friends) also contains capabilities information. We can rename these when the time comes though, I guess.